webapp-django/crashstats/settings/base.py

```diff
@@ -376,12 +346,260 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
+SEARCH_MIDDLEWARE_IMPL = config('SEARCH_MIDDLEWARE_IMPL', 'elasticsearch')
+
+# If you intend to run with DEBUG=False, this must match the URL
+# you're using
+BROWSERID_AUDIENCES = config(
+    'BROWSERID_AUDIENCES',
+    'http://localhost:8000',
+    cast=Csv
+)
+
+# Optional Google Analytics ID (UA-XXXXX-X)
+GOOGLE_ANALYTICS_ID = config('GOOGLE_ANALYTICS_ID', None)
+# Root domain. Required iff you're providing an analytics ID.
+GOOGLE_ANALYTICS_DOMAIN = config('GOOGLE_ANALYTICS_DOMAIN', 'auto')
+
+# Set to True enable analysis of all model fetches
+ANALYZE_MODEL_FETCHES = config('ANALYZE_MODEL_FETCHES', True, cast=bool)
+
+# TODO should we allow overriding this?
+# If you crashstats behind a load balancer, your `REMOTE_ADDR` header
+# will be that of the load balancer instead of the actual user.
+# The solution is to instead rely on the `X-Forwarded-For` header.
+# You ONLY want this if you know you can trust `X-Forwarded-For`.
+# (Note! Make sure you uncomment the line `from . import base` at
+# the top of this file first)
+# base.MIDDLEWARE_CLASSES += (
+#     'crashstats.crashstats.middleware.SetRemoteAddrFromForwardedFor',
+# )
+
+# TODO remove?
+# Ensure that the uploaded file is effectively g+rw while preserving the group
+# ownership inherited from system.
+# See
+# https://docs.djangoproject.com/en/dev/topics/http/\
+#  file-uploads/#changing-upload-handler-behavior
+# FILE_UPLOAD_PERMISSIONS = 0664
+
+# TODO
+# When you don't have permission to upload Symbols you might be confused
+# what to do next. On the page that explains that you don't have permission
+# there's a chance to put a link
+# SYMBOLS_PERMISSION_HINT_LINK = {
+#     'url': 'https://bugzilla.mozilla.org/enter_bug.cgi?product=Socorro&'
+#            'component=General&groups=client-services-security',
+#     'label': 'File a bug in bugzilla'
+# }
+
+
+# Dataservice API configuration
+# Extend dataservices settings from settings/base.py here
+# At a minimum, you'll probably want to change db username/password. All
+# dataservice objects inherit resource configuration and so can all
+# have their database resource configuration set once in 'secrets.postgresql'
+# and 'resource.postgresql' keys.
+DATASERVICE_CONFIG_BASE.update({
+    'secrets': {
+        'postgresql': {
+            'database_password': config(
+                'DATASERVICE_DATABASE_PASSWORD',
+                'aPassword'
+            ),
+            'database_username': config(
+                'DATASERVICE_DATABASE_PASSWORD',
+                'breakpad_rw'
+            ),
+            'database_hostname': config(
+                'DATASERVICE_DATABASE_HOSTNAME',
+                'localhost'
+            ),
+            'database_name': config(
+                'DATASERVICE_DATABASE_NAME',
+                'breakpad'
+            ),
+            'database_name': config(
+                'DATASERVICE_DATABASE_PORT',
+                '5432'
+            ),
+        }
+    }
+})
```

Comment:
> Would be nice to hear what @twobraids has to say about this.

Per the meeting today, sounds like current dataservices impl is going away so not sure it's worth worrying too much about. I think this will tide us over in the meantime.